### PR TITLE
[UX-163] Building on Windows

### PR DESCRIPTION
**Pull request by tfennelly**
Related to issue [UX-163](https://cloudbees.atlassian.net/browse/UX-163) . 

@jenkinsci/code-reviewers  @reviewbybees 

__TODO__:

- [x] Remove JDL symlink and copy assets for Windows
- [x] Fix frontend build in parent pom (see https://github.com/jenkinsci/plugin-pom/pull/26 ).
- [x] Update parent pom dependency after https://github.com/jenkinsci/plugin-pom/pull/26 merge

### CODE REVIEW

**Comment by tfennelly** [comment]
The bablejs polyfill fixed the ES6 errors on IE, but now we're hitting a `react-redux` issue.

After some quick hacking I can see that the `children` object on the following code is failing on a `ReactElement.isValidElement(children)` check because `object.$$typeof === REACT_ELEMENT_TYPE` is `false`. Showing up as a `Symbol`, whatever that means.

````javascript
Provider.prototype.render = function render() {
    var children = this.props.children;

    return _react.Children.only(children);
  };
```

More digging tomorrow.

**Comment by michaelneale** [comment]
@cliffmeyers is there a master configured with write acess to the repo? as something is marking this build as failed but it isn't CI

File: blueocean-web/src/main/js/blueocean.js [hunk 1]
```diff
@@ -1,3 +1,6 @@
+// Apply polyfills before we do anything.
+require('./polyfills');
```
Comment (michaelneale) [inline blueocean-web/src/main/js/blueocean.js]: Is this just for building on windows, or is this related to IE?

File: .gitignore [hunk 2]
```diff
@@ -13,3 +13,4 @@ npm-debug.log
 **/.settings/
 **/.project
 **/.classpath
+blueocean-web/src/main/webapp/
```
Comment (michaelneale) [inline .gitignore]: how come we need this?

**Comment by michaelneale** [comment]
Looking good - I assume still WIP? 

I had a few inline questions. Mostly - is this just for building on windows or is the polyfill for IE support?

File: blueocean-web/src/main/js/blueocean.js (hunk 1 above)
Comment (tfennelly) [inline blueocean-web/src/main/js/blueocean.js]: That's needed to run on IE ... the earlier changes were for building (it should build fine now with `-DskipTests`.

File: .gitignore (hunk 2 above)
Comment (tfennelly) [inline .gitignore]: We just want the folder there for holding the assets, which get put into it at build time. We don't want any of the contents checked in.

**Comment by tfennelly** [comment]
@michaelneale Changing it from a Wip.  Getting this running on IE is not going to be a trivial task, so pushing that part to a separate task for now.

**Comment by michaelneale** [comment]
🐝
